guides/common/modules/proc_configuring-satellite-to-synchronize-content-with-a-local-cdn-server.adoc

```diff
@@ -5,7 +5,7 @@ In a disconnected environment, you must ensure that {ProjectServer} contains the
 
 .Procedure
 
-. Log on to the Red{nbsp}Hat Customer Portal at https://access.redhat.com.
+. Log in to the Red{nbsp}Hat Customer Portal at https://access.redhat.com.
```

Comment:
```suggestion
. Log on to the Red{nbsp}Hat Customer Portal at https://access.redhat.com.
```
IBM Style Guide p. 342:
log in to v
Write as shown - three words. Use only in documentation for UNIX systems, such as AIX. Otherwise, use
“log on to.”

Reply:
We use "log in to" in many other places (found more of them than I found "log on to"), and I think it's appropriate since we do write Linux docs.

Reply:
As @melcorr explained, you log in to a linux system, but log on on some website.

Reply:
The buttons on the Customer Portal itself say "Log in".

Reply:
Docs always say log on, unless referring to the specific UI element? This would break with wider consistency.

Reply:
Would it? 

site:access.redhat.com "log in to the Customer Portal" 
4,050 results

site:access.redhat.com "log on to the Customer Portal"
165 results

Reply:
Lol!! Does 4050 wrongs make a right though? I'm not sure that it does. It's an easy sed also.

Reply:
I mean yeah I definitely think the button on the Customer Portal saying "Log in" is the more compelling of my two arguments haha

Reply:
I'm less inclined to quote IBM style since I have doffed my TW cap, but I do recall a rule that we do not copy over errors from UI elements into the docs. I would argue that the "in" is an error. I would argue that UX really need to start following conventions, because they don't seem to. I would argue that we have a rule and we're not describing the button but the action, and then I would climb back off my soap box and leave you both alone.